jacobian_follower/src/jacobian_follower/jacobian_follower.cpp

```diff
@@ -812,3 +813,46 @@ PointSequence JacobianFollower::get_tool_positions(std::vector<std::string> tool
   std::transform(tool_transforms.cbegin(), tool_transforms.cend(), std::back_inserter(positions), get_translation);
   return positions;
 }
+Eigen::Matrix4Xd JacobianFollower::getLinkToRobotTransform(std::vector<std::string> joint_names,
+                                                           std::vector<double> joint_positions, std::string link_name) {
+  robot_state::RobotState state(model_);
+  state.setVariablePositions(joint_names, joint_positions);
+  auto const &transform = state.getGlobalLinkTransform(link_name);
+  return transform.matrix();
+}
+
+std::vector<Eigen::Matrix4Xd> JacobianFollower::getLinkToRobotTransforms(
+    std::vector<std::string> const &joint_names, std::vector<double> const &joint_positions,
+    robot_state::RobotStatePtr state, std::vector<std::string> const &link_names) const {
+  state->setVariablePositions(joint_names, joint_positions);
```

Comment:
Does this already throw a useful error on invalid input, visible through pybind? If not, let's figure out a way to give the user a hint